src/utils/comparison.ts

```diff
@@ -0,0 +1,9 @@
+export function compareDouble(x: number, y: number) {
+  // const diff = Math.abs(x - y);
```

Comment:
Can you remove the comments here? or is the implementtio within the comments the correct version?

Reply:
Yes! I forgot to clean this up---this epsilon check was breaking some tests so I changed it to the more sensitive direct comparison.